### ISSUE TITLE
Redefinition of `models` method in test scripts

### ISSUE THREAD

**Issue by dmbates** [JuliaStats/MixedModels.jl#691]
Julia v1.10.0-DEV versions are complaining about redefinition of `models` methods in the test scripts
```
Test Summary: | Pass  Total  Time
Missing Omit  |    2      2  0.5s
WARNING: Method definition models(Symbol) in module Main at /Users/dmbates/.julia/dev/MixedModels/test/modelcache.jl:47 overwritten on the same line (check for duplicate calls to `include`).
Test Summary: | Pass  Total  Time
isnested      |    5      5  1.3s
```

Should we wrap the `include` calls for `modelcache.jl` in a test of whether the method exists?

**Comment by palday** [comment]
Yes, that sounds like a good idea to reduce the visual clutter.

**Comment by dmbates** [comment]
Closed by #692
